OpenChange/MAPIStoreMailVolatileMessage.m

```diff
@@ -533,6 +491,48 @@ - (void) getMessageData: (struct mapistore_message **) dataPtr
   return result;
 }
 
+static inline NSString *
```

Comment:
Is there any reason to move the functions around? As it is now, it is difficult to find the differences